### DISCUSSION TITLE
If my AAB is bigger than 150M ,how can i do?

### DISCUSSION THREAD

**Discussion by zengqingyong** [airsdk/Adobe-Runtime-Support#2601] [Q&A]
Before reporting this issue:

- you should consider whether this would be a question better suited to discussions above ^^ ;
- make sure you have searched for similar feature requests and add your ideas there if one exists.

## Description

Describe your feature request in detail. 

- Will this change the current API? How?
- Who will benefit from this feature?

**Comment by marchbold** [comment]
Have a look at Play Asset Delivery:

https://github.com/airsdk/ANE-PlayAssetDelivery

([Documentation](https://github.com/airsdk/ANE-PlayAssetDelivery/wiki))

**Reply by zengqingyong** [reply]
Thank you very much!
